packages/pg/bench.js

```diff
@@ -25,13 +25,11 @@ const exec = async (client, q) => {
 const bench = async (client, q, time) => {
   let start = Date.now()
   let count = 0
-  while (true) {
+  do {
     await exec(client, q)
     count++
-    if (Date.now() - start > time) {
-      return count
-    }
-  }
+  } while (Date.now() - start > time)
```

Comment:
The logic here has been inverted.

Reply:
I don't think the logic has been changed?  just changed to a more natural syntax:
```js
do { ... } while(condition) return x;
```
vs previous
```js
while(true) { ...; if(condition) return x; }
```

Code changes could be minimised by using `// eslint-disable-next-line no-constant-condition` if that's preferred.

Reply:
Exiting a loop when *P* is the opposite of continuing a loop while *P*.

Reply:
:facepalm: of course it is

Reply:
reverted, and added lint exception